backend/src/main/java/com/bakdata/conquery/sql/conversion/cqelement/concept/model/filter/MultiSelectCondition.java

```diff
@@ -0,0 +1,37 @@
+package com.bakdata.conquery.sql.conversion.cqelement.concept.model.filter;
+
+import com.bakdata.conquery.sql.conversion.cqelement.concept.model.FilterCondition;
+import com.bakdata.conquery.sql.conversion.cqelement.concept.model.FilterType;
+import com.bakdata.conquery.sql.conversion.dialect.SqlFunctionProvider;
+import lombok.RequiredArgsConstructor;
+import org.jooq.Condition;
+import org.jooq.Name;
+import org.jooq.impl.DSL;
+
+@RequiredArgsConstructor
+public class MultiSelectCondition implements FilterCondition {
+
+	private final Name column;
+	private final String[] values;
```

Comment:
zu beachten hier ist, dass empty-Strings hier als null/missing-value handle verwendet werden. Also wenn  values `""` oder `null` enthält, will ich explizit _auch_ nach leeren Spalten filtern.

Reply:
```json
"filters":[
  {
    "filter":"geschlecht_select.geschlecht_connector.geschlecht",
    "type":"BIG_MULTI_SELECT",
    "value":[
      "f",
      "",
      null
    ]
  }
]
```

```sql
"concept_0_geschlecht_select_event_filter" as (
    select "pid"
    from "concept_0_geschlecht_select_preprocessing"
    where (
      "concept_0_geschlecht_select_preprocessing"."geschlecht" in (
        'f', ''
      )
      or "concept_0_geschlecht_select_preprocessing"."geschlecht" is null
    )
  )
```
Ok, nochmal zum Verständnis: so wir hier oder nur der NULL check bei leeren Strings und null values?

Reply:
null und "" sind gleich zu behandeln in `value` aber sonst ist der code korrekt